week-3/A-array-find/exercise.js

```diff
@@ -1,13 +1,17 @@
 /* 
-  You are given an array of test scores.
-  Any result over 60 is a pass. 
-  Using .find() (and any other array methods necessary), find the lowest pass score
+  You are given an array of names.
+  Using .find(), we'd like to find the first name which starts with A and is longer than 7 letters.
 */
 
-var testScores = [90, 50, 100, 25, 81, 66, 80];
-var lowestPassScore = findLowestPassScore(testScores);
+// write your code here
+function longNameThatStartsWithA(name) {
+  return name[0].includes("A") && name.length > 7;
```

Comment:
This is correct but you could make a few small improvements.

Firstly when you use `[0]` on a string it will return 1 character. `.includes()` checks if the whole string includes the parameter that you pass to it. But we know that it is just 1 character, so we can just compare directly using `=== "A"`.

The second small problem is what happens if we change the `names` array so that "Alexandra" is changed to "alexandra"? Does it still work? How could we fix that?